projects/demo/src/app/demo/pages/select-page/select-page.component.html

```diff
@@ -78,34 +285,81 @@
         <div class="pa-demo-configuration-field">
             <pa-deprecated-checkbox id="readonly" [(selected)]="readOnlyState">Readonly state</pa-deprecated-checkbox>
         </div>
+        <div class="pa-demo-configuration-field">
+            <pa-deprecated-checkbox id="focused" [(selected)]="firstFocused">focus first select</pa-deprecated-checkbox>
+        </div>
     </pa-demo-configuration>
 
     <pa-demo-usage>
-        <pa-demo-deprecated-base-textfield></pa-demo-deprecated-base-textfield>
+        <h2>Inherited from <code>BaseControl</code></h2>
+        <pa-demo-base-control-usage></pa-demo-base-control-usage>
 
         <br>
         <h2>Specific to <code>SelectComponent</code></h2>
         <h3>Inputs</h3>
         <dl>
+            <dt>acceptHtmlTags</dt>
```

Comment:
yeah... I added this when we decided to add the suggestionMode (user can input free text)

Reply:
I know user can input free text, that's why I'm suggesting to have the internal input with `acceptHtmlTags` to `false` and not allowing developers to set it to `true` on select, because that wouldn't make any sense.

Reply:
yeah... well the only use case I can find is a select were you want to add html tags... like option 1 is <h1> option2 is <h2> ... option5 is <p> and we would not provide <code> so user can input <code>... 
if we refuse that then this should probably be explained to the users of the library...

Reply:
I'm no convinced by your use case at all, that would be a terrible UX! 😆 
You can double check with @ebrehault or @albertcasado if you want, but I really think you should remove that option from the select.

Reply:
it's false by default... I can't see your point... In suggestionMode it behaves like a text-input and this feature is present in all of our text inputs. but I don't really care, I'll just remove it.